routes/web.php

```diff
@@ -24,7 +24,8 @@
 $router->get($uri->translate("case-studies"), [CaseStudiesController::class, "index"])->name("case-studies");
 $router->get($uri->translate("case-studies/{slug}"), [CaseStudiesController::class, "get"])->name("case-studies.entry");
 $router->get($uri->translate("partners"), PartnersController::class)->name("partners");
-$router->get($uri->translate("career"), CareerController::class)->name("career");
+$router->get($uri->translate("career"), [CareerController::class, "index"])->name("career");
+$router->get($uri->translate("career/software-engineer"), [CareerController::class, "softwareEngineerPage"])->name("career.softwareEngineer");
```

Comment:
Upper u have `name("case-studies.entry")`, maybe that will be more consitent:
```suggestion
$router->get($uri->translate("career/software-engineer"), [CareerController::class, "softwareEngineerPage"])->name("career.software-engineer");
```